tests/models/seamless_m4t_v2/test_modeling_seamless_m4t_v2.py

```diff
@@ -835,7 +835,12 @@ def test_generation_languages(self):
     def test_speech_generation(self):
```

Comment:
> tests/models/seamless_m4t_v2test_modeling_seamless_m4t_v2.py::SeamlessM4Tv2GenerationTest::test_speech_generation

previously failed 50% of the time. Now running 100 times and all pass.
(but probably still a bit flaky)